### PR TITLE
Share SqlCredential

### PR DESCRIPTION
**Pull request by Wraith2**
Really easy single file change. Move the file to shared, fixup the docs paths and projects.

### CODE REVIEW

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj [hunk 1]
```diff
@@ -436,19 +436,31 @@
     <Compile Include="Microsoft\Data\SqlClient\SqlColumnEncryptionCngProvider.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlColumnEncryptionCspProvider.cs" />
     <Compile Include="Microsoft\Data\SqlClient\SqlColumnEncryptionEnclaveProvider.cs" />
-    <Compile Include="Microsoft\Data\SqlClient\SqlCommand.cs" />
-    <Compile Include="Microsoft\Data\SqlClient\SqlCommandBuilder.cs" />
+    <Compile Include="Microsoft\Data\SqlClient\SqlCommand.cs">
+      <SubType>Component</SubType>
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj:440]: Please revert "Component" changes"

**Review by cheenamalhotra — APPROVED** [review]
Just a small cleanup needed.
LGTM 👍

File: src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj (hunk 1 above)
Comment (Wraith2) [reply src/Microsoft.Data.SqlClient/netfx/src/Microsoft.Data.SqlClient.csproj]: Done, there's a change in the crypto PR which we think prevents these from being recreated every time the project is opened.

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj [hunk 2]
```diff
@@ -527,7 +527,9 @@
     <Compile Include="Microsoft\Data\SqlClient\SNI\SSRP.cs" />
     <Compile Include="Microsoft\Data\SqlClient\TdsParserStateObjectManaged.cs" />
     <Compile Include="Interop\SNINativeMethodWrapper.Common.cs" />
-    <Compile Include="Microsoft\Data\SqlClient\SqlCredential.cs" />
+    <Compile Include="..\..\src\Microsoft\Data\SqlClient\SqlCredential.cs">
```
Comment (johnnypham) [inline src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:530]: Nit: move to shared section

**Review by johnnypham — APPROVED** [review]

File: src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj (hunk 2 above)
Comment (Wraith2) [reply src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj:530]: Done.

**Pull request merged**